### ISSUE TITLE
Running task after completion of other tasks

### ISSUE THREAD

**Issue by AnanthRavi** [fabric/fabric#1426]
Hey there,
I was wondering if its possible to run a task at the end. Something like this

```
fab hosts deploy
```

```python
deploy():
  task1()
  task2()
  task3()
```

Is it possible to run just task3 after task 1 and 2 have been run on all hosts?

**Comment by kaaveland** [comment]
Yes, this is possible. Invoking as `fab task1 task2 task3` will do just that. You can also structure it as you have written the code above, but you'll need to use `execute` to get the desired behaviour.

You can find some inspiration here:

https://github.com/fabric/fabric/issues/1414

If you need to read up on `execute`, this is a good place to start: http://docs.fabfile.org/en/1.10/usage/execution.html
